no-computer.md

```diff
@@ -11,41 +11,93 @@ _What need is met by your idea? Who are you helping? What is that person's
+interact in such a way that their story can be constructed from one
+of exponentially many combinations of modular story blocks.
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+The "need" is simply a desire for entertainment, so if the project
+manages to entertain the user, it has served its purpose.  Language is
+a natural choice for any kind of storytelling, and we need a language
+that is specific to our domain of precise language to be able to construct
+stories logically guaranteed to reach some conclusion.
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+I was looking at a book and thinking about how it was linear, but there
+do exist books with "create your own ending" type stories. This was when
+I imagined if we were able to pick not just between two endings, which
+arrive through our choices, but rather construct our own endings using
+myriad modular building blocks?
 
 ### Domain
 _Describe the project's domain in five words._
 
+Creating chaotic stories without planning.
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain?_ 
 
+The user might have a deck of cards from which they can construct the initial
+state of their character(s), setting(s), and plots.  From this point on, the
+user might use a booklet or story guide which uses information about the
+items or number of characters found in a story to determine the next step,
+these choices build on one another, leading to a chaotic sequence of next
+steps highly dependent upon initial conditions.
+
+For example, one rule might read:
+"If at least one character has been injured, and has not yet healed, add
+the next tile from set #42."
 
 ### Operation (semantics)
 _What might happen when a program runs? How does a program interact with the
 user? What kinds of errors might occur, and how might they be communicated to
 the user?_
 
+The running of the program is the repeated lookup (by the user) of the next
+step in the story.  Each next step should have some relation to all of the
+the rules that can produce it so that the story, while chaotic, does not just
+read as a pseudorandom hodgepodge of events, but rather a silly and loosely
+connected thread of continuity.
+
+Errors would result on the behalf of the user following with an incorrect
+card, which they would either discover on their own, or not.  Either way
+is acceptable.
 
 ### Expressiveness
 _What should be easy to do in this language? What should be possible, but
 difficult? What should be impossible or very difficult?_
 
+It should be easy to generate a silly story based on simple inputs.
+
+It should be difficult to predict the next step in the story.
+
+It should be impossible to predict the ending of the story given the
+starting configuration.
```

Comment:
nice point! there would need to be a sufficient number of branches for the way the story can change.